src/view/ConversationWindow.vala

```diff
@@ -288,10 +292,15 @@ namespace Venom {
     }
 
     public signal void send();
-
+    public signal void insert_new_line();
+    
     public bool on_key_press_event(Gdk.EventKey event) {
-      if (event.keyval == key.accel_key && event.state == key.accel_mods) {
-        send();
+      if (event.keyval == key.accel_key) {
+        if ((event.state & Gdk.ModifierType.SHIFT_MASK) > 0) {
+          insert_new_line();
```

Comment:
Please remove the `insert_new_line` and just return false, otherwise we are catching stuff here that we aren't supposed to. If this will be needed in the future we can always add it.